docs/pages/docs/apis/config.mdx

```diff
@@ -396,29 +396,54 @@ See the [schema extension guide](../guides/schema-extension) for more details on
 import type { StorageConfig } from '@keystone-6/core/types'
 ```
 
-The `storage` config option is used to provide information for use by the [`file`](./fields#file) field type or the
-[`image`](./fields#image) field type. You provide an object whose each property is a `StorageConfig` object, which fields may then reference.
+The `storage` config option provides configuration which is by the [`file`](./fields#file) field type or the
+[`image`](./fields#image) field type. You provide an object whose property is a `StorageConfig` object, fields then reference this `storage` by the key.
 Each storage is configured separately using the options below.
 
-A single storage may be used by multiple file or image fields.
-
-When using the image or file field, you use the key from the object to specify which storage to use.
+A single storage may be used by multiple file and/or image fields.
 
 Options:
 
-- `type`: This sets whether this storage is used for storing files or images. Can be either `"file"` or `"image"`.
-- `kind`: Whether the storage will be on the machine, or in an [s3 bucket](https://aws.amazon.com/s3/) `"local"` or `"s3"`
-- `preserve`: Set whether the items should be deleted at the source when they are removed from Keystone's database. We
-  recommend that you set `preserve: false` unless you have a strong reason to preserve files that Keystone cannot reference.
+- `kind`: Whether the storage will be on the machine `"local"`, or in an [s3 bucket](https://aws.amazon.com/s3/) `"s3"`
+- `preserve`: Defines whether the items should be deleted at the source when they are removed from Keystone's database. We
+  recommend that you set `preserve: false` unless you have a strong reason to preserve files that Keystone cannot reference. The
+  default is `false`.
 
 Local options:
 
 - `generatedUrl`: A function that recieves a partial path with the filename and extension, and the result
-  of which will be used as the `url` in the field's graphql - this should point to where the item is hosted.
+  of which will be used as the `url` in the field's graphql - this should point to where the client can retrieve the item.
 - `serverRoute`: Sets whether or not to add a server route to Keystone. Set it to `null` if you don't want
   keystone to host the images, otherwise set it to an object with a `path` property
   - `path`: The partial path where keystone will host the images, eg `/images` or `/files`
 
+```typescript
+import { config } from '@keystone-6/core';
+import { lists } from './schema';
+
+export default config({
+   /* ... */
+  storage: {
+    my_images: {
+      kind: 'local',
+      generatedUrl: path => `http://localhost:3000/images${path}`,
+      serverRoute: {
+        path: '/images',
+      },
+      storagePath: 'public/images',
+    },
+    my_files: {
```

Comment:
@borisno2 you good if we only provide examples of one storage option for this just to reduce code length or do you think having two is important?

Reply:
Hey @Noviny
Yeah Maybe we can cut this down a bit (ie one example of each `kind` ?) We can then expand on this in a more in-depth guide. I'll have a go at cutting this down this morning.

Reply:
Just pushed a new example @Noviny with both 'kind`s in one example - do you think makes sense?